### PR TITLE
Problem matchers in the correct file

### PR DESCRIPTION
**Pull request by Stevendeo**

### CODE REVIEW

**Comment by Stevendeo** [comment]
This should add what was missing in https://github.com/OCamlPro/superbol-studio-oss/pull/83 @GitMensch

File: package.json [hunk 1]
```diff
@@ -245,4 +237,4 @@
     "Snippets",
     "Other"
   ]
-}
+}
```
Comment (GitMensch) [inline package.json:240]: somehow this missing NL gets in when creating a file!

**Comment by GitMensch** [comment]
Would you mind sending those commits directly "upstream"? No need to include my commit up-front.

**Comment by Stevendeo** [comment]
Okay, I'm closing this one

**Pull request closed without merging**